sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1440,6 +1446,7 @@ private void SetNameFieldsIfNull()
             IProgress<long> progressHandler,
             string operationName,
             bool async,
+            bool? userPrincipalName,
```

Comment:
NIT: traditionally we have the `async` and `cancellationToken` as the last parameters.